### PR TITLE
PEOPLE app using microservices

### PR DESCRIPTION
**Pull request by lhsvobodaj**
The solution for the proposed PoC test.

### CODE REVIEW

**Comment by vinnylinck** [comment]
Reviewing the code. Thanks a lot, man. :)

File: node/app/model/people.js [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+    //Promise to enqueue data in RabbitMQ
+    let rabbitmq = new Promise((resolve, reject) => {
+      Rabbit.send(JSON.stringify(people), () => {
+        resolve('success');
+      });
+    });
+
+    //Promise to persist data in Elasticsearch
+    let elasticsearch = new Promise((resolve, reject) => {
```
Comment (lhsvobodaj) [inline node/app/model/people.js]: @vinnylinck Is still possible to modify a bit this code? I would like to make a test using the elastic rabbitmq plugin. Once the plugin has been configured, I should be able to detach the elastic.persist step from the main application.

Comment (vinnylinck) [inline node/app/model/people.js]: Be my guest, man.

**Comment by vinnylinck** [comment]
I finish my review. The HR is going to contact you informing about the results and next steps. Thanks a lot.

**Pull request closed without merging**